package.json

```diff
@@ -5,20 +5,27 @@
     "node": ">=14"
   },
   "scripts": {
-    "postversion": "git push --tags",
     "build:development": "cross-env RAILS_ENV=development NODE_ENV=development ./bin/webpack",
     "build:production": "cross-env RAILS_ENV=production NODE_ENV=production ./bin/webpack",
+    "fix:js": "yarn lint:js --fix",
+    "fix:json": "prettier --write \"**/*.json\"",
+    "fix:md": "prettier --write \"**/*.md\"",
+    "fix:sass": "stylelint --fix \"**/*.{css,scss}\" && prettier --write \"**/*.{css,scss}\"",
+    "fix:yml": "prettier --write \"**/*.{yaml,yml}\"",
+    "fix": "yarn fix:js && yarn fix:json && yarn fix:sass && yarn fix:yml",
+    "jest": "cross-env NODE_ENV=test jest",
+    "lint:js": "eslint . --ext=.js,.jsx,.ts,.tsx --cache --report-unused-disable-directives",
+    "lint:json": "prettier --check \"**/*.json\"",
+    "lint:md": "prettier --check \"**/*.md\"",
+    "lint:sass": "stylelint \"**/*.{css,scss}\" && prettier --check \"**/*.{css,scss}\"",
+    "lint:yml": "prettier --check \"**/*.{yaml,yml}\"",
+    "lint": "yarn lint:js && yarn lint:json && yarn lint:sass && yarn lint:yml",
     "manage:translations": "node ./config/webpack/translationRunner.js",
+    "postversion": "git push --tags",
+    "prepare": "husky install",
     "start": "node ./streaming/index.js",
-    "test": "${npm_execpath} run test:lint:js && ${npm_execpath} run test:typecheck && ${npm_execpath} run test:jest",
-    "test:lint": "${npm_execpath} run test:lint:js && ${npm_execpath} run test:lint:sass",
-    "test:lint:js": "eslint --ext=.js,.jsx,.ts,.tsx . --cache --report-unused-disable-directives",
-    "test:lint:sass": "stylelint \"**/*.{css,scss}\" && prettier --check \"**/*.{css,scss}\"",
-    "test:typecheck": "tsc --noEmit",
-    "test:jest": "cross-env NODE_ENV=test jest",
-    "format": "prettier --write .",
-    "format-check": "prettier --check .",
-    "prepare": "husky install"
+    "test": "yarn test:lint && yarn run test:typecheck && yarn test:jest",
+    "test:typecheck": "tsc --noEmit"
```

Comment:
Not sure if the `test:` prefix should also be dropped here

Reply:
IMO it should be dropped, or changed to `lint:typescript`, as this is more a lint than a test.